libraries/RC_Channel/RC_Channel_aux.h

```diff
@@ -91,6 +91,9 @@ class RC_Channel_aux : public RC_Channel {
         k_rcin14                = 64,
         k_rcin15                = 65,
         k_rcin16                = 66,
+        k_ignition              = 67,
```

Comment:
Ah, this issue has been fixed in master with this commit:  https://github.com/ArduPilot/ardupilot/commit/7ea68f8f804189c686d3704f55324e7bc6bfe897

Looks ok to me.